coderd/users_test.go

```diff
@@ -1677,6 +1777,17 @@ func TestUsersFilter(t *testing.T) {
 					exp = append(exp, made)
 				}
 			}
+
+			// TODO: This can be removed with dbmem
+			if !dbtestutil.WillUsePostgres() {
```

Comment:
The plan is to eventually get rid of `dbmem` entirely, so I'm not too worried about cluttering up this test. This is to handle a disparity between organization-less members in Postgres and dbmem. (FWIW it's not possible to have organization-less members on a real Coder deployment.)